<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpgradeWin</class>
 <widget class="QWidget" name="UpgradeWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upgrade</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Node</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nodeComboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 1. Upgarde Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 2. Send File</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="upgradeProgressBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="fpgaRadioButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FPGA Upgrade</string>
   </property>
  </widget>
  <zorder>resetButton</zorder>
  <zorder>label</zorder>
  <zorder>nodeComboBox</zorder>
  <zorder>enterButton</zorder>
  <zorder>browseButton</zorder>
  <zorder>fileEdit</zorder>
  <zorder>sendButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>upgradeProgressBar</zorder>
  <zorder>fpgaRadioButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
